node/src/rpc.rs

```diff
@@ -51,7 +52,9 @@ where
 	let FullDeps { client, pool, deny_unsafe } = deps;
 
 	io.extend_with(SystemApi::to_delegate(FullSystem::new(client.clone(), pool, deny_unsafe)));
-	io.extend_with(TransactionPaymentApi::to_delegate(TransactionPayment::new(client)));
-
+	io.extend_with(TransactionPaymentApi::to_delegate(TransactionPayment::new(client.clone())));
+	io.extend_with(pallet_messages_rpc::MessagesApi::to_delegate(
+		pallet_messages_rpc::MessagesHandler::new(client.clone()),
+	));
```

Comment:
Should we group out mrc related handlers here, since schema and other pallets will expose rpc

Reply:
Sounds good. Going to apply for additional RPC